packages/cli/src/cmds/validator/signers/importExternalKeystores.ts

```diff
@@ -37,9 +37,6 @@ export async function readPassphraseOrPrompt(args: {importKeystoresPassword?: st
       },
     ]);
 
-    // eslint-disable-next-line no-console
-    console.log("Password is correct");
```

Comment:
This console.log is incorrect and not helpful as password verification is done later in the process